docs/core/whats-new/index.md

```diff
@@ -0,0 +1,160 @@
+
+- The `Main` method, the application entry point, can be marked with the [async](../../csharp/language-reference/keywords/async.md) keyword.
+- Inferred tuple names.
+- Default expressions.
+- Pattern matching with generics. 
+
+<!-- For more information see [link to C# what's new](url). -->
+
+## Platform improvements
+
+.NET Core 2.0 includes a number of features that make it easier to install .NET Core and to use it on supported operating systems.
+
+### .NET Core for Linux is a single implementation**
+
+.NET Core 2.0 offers a single Linux implementation that works on multiple Linux distributions. .NET Core 1.x required that you download a distribution-specific Linux implementation.
+
+You can also develop apps that target Linux as a single operating system. .NET Core 1.x required that you target each Linux distribution separately. 
+
+### Support for the Apple cryptographic libraries
+
+.NET Core 1.x on macOS required the OpenSSL toolkit's cryptographic library. .NET Core 2.0 uses the Apple cryptographic libraries and doesn't require OpenSSL, so you no longer need to install it. 
+
+## API changes and library support
+
+### Support for .NET Standard 2.0
+
+The .NET Standard defines a versioned set of APIs that must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. .NET Core 1.x supports the .NET Standard version 1.6; .NET Core 2.0 supports the latest version, .NET Standard 2.0.  For more information, see [.NET Standard](../../standard/net-standard.md).
+
+The .NET Standard 2.0 includes over 20,000 more APIs than were available in the .NET Standard 1.6. Much of this expanded surface area results from incorporating APIs that are common to the .NET Framework and Xamarin into .NET Standard.
+
+.NET Standard 2.0 class libraries can also reference .NET Framework class libraries, provided that they call APIs that are present in the .NET Standard 2.0. No recompilation of the .NET Framework libraries is required.
+
+For a list of the APIs that have been added to the .NET Standard since its last version, the .NET Standard 1.6, see [.NET Standard 2.0 vs. 1.6](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md).
+
+### Expanded surface area
+
+The total number of APIs available on .NET Core 2.0 has more than doubled in comparison with .NET Core 1.1. 
+
+### Support for .NET Framework libraries
+
+.NET Core code can reference existing .NET Framework libraries, including existing NuGet packages. Note that the libraries must use APIs that are found in .NET Standard.
+
+## Visual Studio integration
+
+Visual Studio 2017 version 15.3 and in some cases Visual Studio for Mac offer a number of significant enhancements for .NET Core developers.
```

Comment:
Consider removing "in some cases".  If the cases aren't identified, this doesn't provide much value.